file_sync_server/main.py

```diff
@@ -8,20 +8,23 @@
 
 app = FastAPI()
 
+class FileNames(BaseModel):
+    filename: list
```

Comment:
I feel like it's sorta weird to call a list of filenames "filename" singular--would "names" make more sense?